pkg/daemonsetlock/daemonsetlock.go

```diff
@@ -79,6 +86,11 @@ func (dsl *DaemonSetLock) Test(metadata interface{}) (holding bool, err error) {
 		if err := json.Unmarshal([]byte(valueString), &value); err != nil {
 			return false, err
 		}
+
+		if ttlExpired(value.Created, value.TTL) {
+			return true, nil
```

Comment:
I think I would print a log message in this case, for when it does something unexpected.

Reply:
I can not see any other log references inside this pkg, shall this be changed then?

Reply:
That's valid; could add another return value and print at the higher level?

Reply:
I do not think I could find place for `err` inside `ttlExpired` function as this is simple true/false check function.

If I'd return an err value here, it would be errored as fatal - https://github.com/weaveworks/kured/blob/f2a0f8e20dbe6b0cc4a5d03cbe06889333adee98/cmd/kured/main.go#L196-L205

Buuut, I might be blind...

Reply:
OK I'll fix it in post.